src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -303,6 +304,10 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('web_link.xml');
         }
 
+        $loggerDefinition = $container->register(LoggerInterface::class, Logger::class);
```

Comment:
Shouldn't it be in a pass and only if there is no `logger` service/alias already registered? (in other words, won't it conflict with the MonologBundle depending on the order the bundles are registered in the kernel? FrameworkBundle usually come first, but still)

Reply:
The idea is indeed to provide this simple logger only when monolog (or any other logger) is not registered. Even if Flex tries to keep FrameworkBundle first, we should not rely on this.